index.html

```diff
@@ -1,113 +1,177 @@
 
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css?family=Rubik:700" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
-    
-    <!--========= My Style Sheet =======-->
-    <link rel="stylesheet" href="assets/css/style.css">
-</head>
-<body id="page-top">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Rubik:700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
 
-
-    <nav class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase navbar-shrink" id="mainNav">
-        <div class="container">
-            <img class="navbar-brand" src="assets/images/school.png" alt="logo">
-            <a class="navbar-brand js-scroll-trigger" href="#page-top">BACKPACK.com</a>
-            <button class="navbar-toggler navbar-toggler-right text-uppercase bg-primary text-white rounded"
-                type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive"
-                aria-expanded="false" aria-label="Toggle navigation">
-                Menu
-                <i class="fas fa-bars"></i>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarResponsive">
-                <ul class="navbar-nav ml-auto">
-                    <li class="nav-item mx-0 mx-lg-1">
-                        <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#home">Back to
-                            Top</a>
-                    </li>
-                    <li class="nav-item mx-0 mx-lg-1">
-                        <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about">About Us</a>
-                    </li>
-                    <li class="nav-item mx-0 mx-lg-1">
-                        <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#contact">Contact Us</a>
-                    </li>
-                </ul>
-            </div>
+    <!--========= My Style Sheet =======-->
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body id="page-top">
+    <nav
+      class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase navbar-shrink"
+      id="mainNav"
+    >
+      <div class="container">
+        <img class="navbar-brand" src="assets/images/school.png" alt="logo" />
+        <a class="navbar-brand js-scroll-trigger" href="#page-top"
+          >BACKPACK.com</a
+        >
+        <button
```

Comment:
Great job but I don't think that HTML should be formatted this way this looks more like css.